buildtools/windows_inno_installer.iss

```diff
@@ -3,14 +3,15 @@
 
 #define MyOrgName "Tari"
 #define MyAppName "Base Node"
-#define MyAppVersion "0.2.1-3435d4c-release"
+#define MyAppVersion "0.2.1-23677a7-release"
```

Comment:
Yes @CjS77, this is a bit of a manual task. The approach followed currently that ensures the version number is correct is as follows:

- checkout the identified commit that needs to be released, do a  `tari_base_node.exe` release build
- checkout the latest development branch, create a new Windows runtime installer branch
- change the Inno build script to reflect the correct version number and any other install or runtime changes that do not affect the release build, like environment settings, etc.
- create the _Windows installer_ using the runtime changes and Inno build script - note this will pull the `tari_base_node.exe` release build created earlier with the identified commit
- create a PR and share the _Windows installer_ somewhere where the reviewers can get hold of it, fix all review changes, updating the _Windows installer_ as required 

When the PR (and thus _Windows installer_) is approved, share the _Windows installer_ with the supporters.